.ahoy.yml

```diff
@@ -33,6 +33,11 @@ commands:
   up:
     usage: Build and start Docker containers.
     cmd: |
+      # Log into to Github docker registry if custom MYSQL_IMAGE is set.
+      if [ "$MYSQL_IMAGE" ]; then
+        GITHUB_USER=$(curl -sSL -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/user | jq -r .login)
```

Comment:
Potentially could add some validation here for the existence of `GITHUB_TOKEN` but if this blows up I think its fairly obvious what happened as things stand.